visidata/vd.py

```diff
@@ -1752,6 +1746,12 @@ def __setitem__(self, k, v):
 
 options = OptionsObject(base_options)
 
+# Generator => A .. Z AA AB ...
+chars = string.ascii_uppercase
```

Comment:
Don't create this at the global scope.  It's only used once in the following generator, so just use string.ascii_uppercase there.